swagger/v1/swagger.json

```diff
@@ -61,13 +59,14 @@
               "application/json": {
                 "resources": [
                   {
+                    "updated_at": "2021-08-12T03:11:24.138Z",
                     "alternate_name": null,
                     "certified": false,
                     "email": null,
                     "id": 68,
                     "legal_status": null,
                     "long_description": null,
-                    "name": "Mosciski-Bode",
+                    "name": "Greenfelder and Sons",
```

Comment:
We might want to make the data fixtures deterministic for the Swagger stuff, or else every time we regenerate them, we're going to get a ton of differences.